config/routes.rb

```diff
@@ -1153,7 +1153,8 @@
     scope module: :vendor_api_docs, path: '/api-docs' do
       get '/' => 'pages#home', as: :home
       get '/usage-scenarios' => 'pages#usage', as: :usage
-      get '/reference' => 'reference#reference', as: :reference
+      get '/reference' => redirect("/api-docs/v#{VendorAPI::VERSION}/reference"), as: :reference
```

Comment:
⭐

Reply:
This could result in redirecting to the sandbox released version when in production. Is that the expected behavior or do we need some logic to retrieve the production available version?